enable/example_application.py

```diff
@@ -0,0 +1,75 @@
+#
+# (C) Copyright 2015 Enthought, Inc., Austin, TX
+# All right reserved.
+#
+# This file is open source software distributed according to the terms in
+# LICENSE.txt
+#
+"""
+Example Application Support
+===========================
+
+This module provides a simple Pyface application that can be used by examples
+in places where a DemoFrame is insufficient.
+
+"""
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+
+from pyface.api import ApplicationWindow, GUI
+
+class DemoApplication(ApplicationWindow):
```

Comment:
Actually, I think this class is named correctly now that I've seen the example code. I think my problem is the `window = cls(**traits)` line in `demo_main`. That reads a little strangely.